unicorecmsffl/templates/search_results.pt

```diff
@@ -0,0 +1,24 @@
+<div metal:use-macro="view.global_template"
+    xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+    i18n:domain="unicorecmsffl">
+    <div metal:fill-slot="content">
+        <div>
+            <div tal:condition="paginator" class="box item">
+                <p tal:condition="paginator.total_count()==1"><span tal:omit-tag="" i18n:translate='1 search result for:'>1 search result for: </span><b>${query}</b></p>
+                <p tal:condition="paginator.total_count() > 1" >${paginator.total_count()} <span tal:omit-tag="" i18n:translate='search results for:'> search results for: </span><b> ${query}</b></p>
+            </div>
+
+            <div metal:use-macro="view.paginator_template"></div>
+            </div>
+                <div tal:repeat="result paginator.get_page()" tal:condition="paginator" class="box item">
+                        <p tal:condition="result.primary_category">${view.get_category(result.primary_category).title}</p>
+                        <a href="/content/detail/${result.uuid}/">${result.title} ...</a>
+                        <a href="/content/detail/${result.uuid}/" i18n:translate='Read More'>Read More &gt;</a>
```

Comment:
`&gt;` should be in a separate tag because when the text is translated, `Read More` will replace the whole text block.
What you need is something like `<a href=""><span tal:omit-tag="" i18n:translate='Read More'>Read More</span> &gt;</a>